<ui version="4.0" >
 <class>Pakkorn</class>
 <widget class="QMainWindow" name="Pakkorn" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>pakkorn</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../ressources/pakkorn-02.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="searchFilter" >
          <property name="editable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="categoriesWidget" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy" >
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy" >
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="tabKeyNavigation" >
           <bool>true</bool>
          </property>
          <property name="dragEnabled" >
           <bool>true</bool>
          </property>
          <property name="iconSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="textElideMode" >
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="verticalScrollMode" >
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode" >
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="uniformItemSizes" >
           <bool>true</bool>
          </property>
          <property name="sortingEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="detailWidget" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>D</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>I</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>L</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Full Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Version</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>7</number>
        </property>
        <property name="textVisible" >
         <bool>false</bool>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance" >
         <bool>true</bool>
        </property>
        <property name="textDirection" >
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>91</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="push_update" >
        <property name="text" >
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_download" >
        <property name="text" >
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_install" >
        <property name="text" >
         <string>Install</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>19</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::TabFocus</enum>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_2" />
    <addaction name="action_Close" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_website" />
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Update" />
    <addaction name="action_Install" />
    <addaction name="separator" />
    <addaction name="action_Show" />
    <addaction name="separator" />
    <addaction name="action_Configuration" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Edit" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Close" >
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Open_2" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Update" >
   <property name="text" >
    <string>&amp;Update</string>
   </property>
  </action>
  <action name="action_Install" >
   <property name="text" >
    <string>&amp;Install</string>
   </property>
  </action>
  <action name="action_Show" >
   <property name="icon" >
    <iconset>../../../../../../../../../../usr/lib/Qt/4.2.3/tools/designer/src/components/formeditor/images/submenu.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Show</string>
   </property>
  </action>
  <action name="action_website" >
   <property name="text" >
    <string>&amp;website</string>
   </property>
  </action>
  <action name="action_Configuration" >
   <property name="text" >
    <string>&amp;Configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>categoriesWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Pakkorn</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>150</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_update</sender>
   <signal>clicked()</signal>
   <receiver>Pakkorn</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>570</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel" >
     <x>373</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
